db/migrate/20170607101716_create_themes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateThemes < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
I hadn't. I forgot those existed.

Do we have a use-case other than Themes and Subthemes for grouping content items?